tools/tsconfig.json

```diff
@@ -8,8 +8,10 @@
     "noUncheckedIndexedAccess": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "strict": true
+    "strict": true,
+    "outDir": "dist"
   },
+  "include": ["*"],
```

Comment:
Why can we just do that here but not in e.g. native?

Reply:
in tools there is only a flat list of files, e.g. `next-version.ts`, `manage-metadata.ts`. In native we have a directory and additionally some other files, therefore the glob becomes more complicated. But in short it should be prefered to use more specific globs, as (especially in native) a lot of irrelevant nested directories would be scanned for ts/tsx/js/.. files. This is definitely discouraged.